### ISSUE TITLE
Enable the admin to subscribe to the GlobaLeaks newsletter during activation wizard

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/GlobaLeaks#1855]
In reference to implementation of GlobaLeaks Newsletter at #1854 , this ticket is to enable the admin of a GlobaLeaks site to subscribe from the Activation Wizard (during the initial setup of the platform) to subscribe to the Newsletter.

By default the admin should be subscribed to the newsletter, during the activation wizard the admin can opt-out not to do it.

**Comment by NSkelsey** [comment]
@fpietrosanti we are looking at this now with the idea to bundle a few wizard changes in one merge, specifically #1927 and #2019.,

 If you can setup the newsletter side, I can put in the sign up right quick.

**Comment by evilaliv3** [comment]
Here before an implementation could happen i think we should fist discuss few points:
1) which technology do we want to use for the newsletter and implement it
2) do we want to couple globaleaks newsletter implementation to the newsletter facility of maybe it is better to handle this process via a "globaleaks server facility" api like described in https://github.com/globaleaks/GlobaLeaks/issues/1849

both the abbroaches for 2. have pro/cons; i would probably prefer the second because the underlying technology for the newsletter may change (for our choice) or for the provider choice while the more globaleaks will be used is possible that a "globaleaks provider" could have pre-configured VPS with a specific version of globaleaks that will statically continue to point to an old newsletter implementation.

for this and other reasons i would suggest to postpone the implementation for a while.

**Comment by NSkelsey** [comment]
@evilaliv3 frankly, I would suggest our existing infrastructure and avoid complicating something that is simple.

However it looks like the mailing server is having a bad week:

```bash
~$ curl http://lists.globaleaks.org/mailman/listinfo
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="http://lists.globaleaks.org/mailman/listinfomailman/listinfo">here</a>.</p>
</body></html>
```

The suggestion in the other thread is to use mail chimp https://kb.mailchimp.com/lists. If the mailing list is opt-in the only thing anybody is getting is more spam.

**Comment by evilaliv3** [comment]
yep i may agree, i was just braindumping the possible concerns and postponing the implementation of the feature inside globaleaks to an actual decision of the newsletter technology.

**Comment by fpietrosanti** [comment]
But we don't have a newsletter running actually, it would first require to be implemented on our website and have content on it, so it's now not that much useful. Admin would instead need to get notice of new globaleaks releases
